src/System/Tasks/ConversionRunner.ts

```diff
@@ -489,6 +493,36 @@ export class ConversionRunner
             };
         }
 
+        /**
+         * Add Markdown IT Container
+         */
+        var markdownContainerOptions = {
+            validate: function (name: string) {
+                return name.trim().length;
+            },
+            render: function (token:Token[], id:number) {
+                if (token[id].info.trim() !== '') {
+                    return `<div class="${token[id].info.trim()}">\n`;
+                } else {
+                    return `</div>\n`;
+                }
+            }
+          };
+
+        parser.use(markdownContainer,'',markdownContainerOptions);
+
+        /**
+         * Add Markdown IT Include
+         */
+
+        var markdownIncludeOptions = {
+            root: path.dirname(window.activeTextEditor.document.fileName),
+            includeRe: /\!{3}\s*include(.+?)\!{3}/i
+          };
+
+        parser.use(markdownInclude, markdownIncludeOptions);
```

Comment:
Please do not use the `var` keyword. Instead just pass the options directly:
```suggestion
        parser.use(
            markdownContainer,
            '',
            {
                validate: function (name: string) {
                    return name.trim().length;
                },
                render: function (token:Token[], id:number) {
                    if (token[id].info.trim() !== '') {
                        return `<div class="${token[id].info.trim()}">\n`;
                    } else {
                        return `</div>\n`;
                    }
                }
          });

        /**
         * Add Markdown IT Include
         */
        parser.use(
            markdownInclude,
            {
                root: path.dirname(window.activeTextEditor.document.fileName),
                includeRe: /\!{3}\s*include(.+?)\!{3}/i
            });
```

Reply:
Commit Resolved: **PR Modifications ConversionRunner TS** ( 7d51275bb9492a485000963129e1df40a3f7b6ca )